cibyl/exceptions/model.py

```diff
@@ -21,7 +23,7 @@
 class NonSupportedModelType(CibylException):
     """Exception for trying to populate non-supported model"""
 
-    def __init__(self, model_type):
+    def __init__(self, model_type: object):
```

Comment:
More than 'object', wouldn't the type be 'type' here?

Reply:
Yes, 'type' is better here. Thank you!